scripts/setup.sh

```diff
@@ -10,7 +10,7 @@ export PATH="${PATH}:$(yarn global bin)"
 
 # Install required node version
 export NVM_DIR="/root/.nvm"
-NODE_VERSION="${1:-v12.20.0}"
+NODE_VERSION="${1:-v14.19.3}"
```

Comment:
what is the dependency that requires this specific min version?

Reply:
auth-js